integration/solana/call_anchor.spec.ts

```diff
@@ -0,0 +1,23 @@
+import expect from 'expect';
+import { loadContract } from './setup';
+import { publicKeyToHex } from '@solana/solidity';
+import { SystemProgram, PublicKey, Keypair } from '@solana/web3.js';
+
+
+describe('Deploy solang contract and test', function () {
```

Comment:
We have many tests named as `Deploy slang contract and test`. We should avoid using this name all over again, because it is not descriptive of what you are testing.